src/routes/health/health.spec.ts

```diff
@@ -0,0 +1,28 @@
+import { handler as healthCheck, Health, setRouteHealth } from '.';
+import { Request, Response } from 'express';
+
+describe('API Routing Health Check', () => {
+  let req: Partial<Request> ;
+  let res: Partial<Response>;
+  let status: Health;
+
+  beforeAll(() => {
+    res = {
+      send: jest.fn(),
+      status: jest.fn()
+    };
+    req = {};
+  });
+
+  it('sets route health', () => {
+    status = Health.busy;
+    req.path = '/init';
+    setRouteHealth(status, <Request>req);
+  });
+
+  it('returns a response', () => {
+    healthCheck(<Request>req, <Response>res);
+    expect(res.send).toBeCalledTimes(1);
+    expect(res.status).toHaveBeenCalledWith(200);
```

Comment:
We should `expect.assertions(2)` here.

Reply:
This has been resolved in the latest changes